articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
+In order to call the endpoints of [Auth0 Management API v2](/api/management/v2), you need a token, what we refer to as Auth0 Management APIv2 Token. This token is a [JWT](/jwt), contains specific granted permissions (known as __scopes__), and is signed with a client API key and secret for the entire tenant.
 
-## How to get a Management APIv2 Token
+There are two ways to get a Management APIv2 Token: [get one manually using the Dashboard](#get-a-token-manually), or [automate the process](#automate-the-process) (build a simple command line tool that generates tokens). In this article we will see how you can do either.
 
-**Create a Non Interactive Client**
+## Get a token manually
 
-A Non Interactive Client is a client that interacts with an API where there is no user involved. It's a machine to machine interaction. This must be used instead of a Single Page or Native apps because those cannot meet the necessary security requirements for executing this type of flow.
+<div class="alert alert-info">
+  <strong>Heads up!</strong> Before you decide to follow the manual process, you should know that the Management APIv2 token has a validity of <strong>24 hours</strong>. After that the token will expire and you will have to get a new one. If this doesn't work for you, you can <a href="#automate-the-process">automate the process</a>.
+</div>
 
-To create a new Non Interactive Client, go to the [Clients section](${manage_url}/#/clients) of the dashboard then click the **CREATE CLIENT** button.
+Let's see how you can get a token manually. Note, that the first two steps of the process need to be executed _only_ the first time. This might be true also for the third step, if the endpoints you need to access do not change. In that case you can skip to the [Get the Token](#4-get-the-token) section.
 
-Enter a name for your new client, select **Non Interactive Clients** and then click the **CREATE** button.
+### 1. Create a Client
+
+First, you need to create a Non Interactive Client. If you already have one skip this paragraph.
+
+  ::: panel-info What is a Non Interactive Client?
+  A Non Interactive Client represents a program that interacts with an API where there is no user involved. An example would be a server script that would be granted access to consume a Zip Codes API. It's a machine to machine interaction. This must be used instead of a Single Page or Native apps because those cannot meet the necessary security requirements for executing this type of flow. If you want to read more about calling APIs this way, refer to [Calling APIs from a Service](/api-auth/grant/client-credentials).
+  :::
+
+Go to the [Clients section](${manage_url}/#/clients) of the dashboard and click the **Create Client** button.
```

Comment:
Can we recommend using a descriptive name for the client, like "Auth0 Management API Client"?